packages/datadog-plugin-redis/src/index.js

```diff
@@ -73,6 +66,11 @@ class RedisPlugin extends Plugin {
       span.finish()
     })
   }
+
+  configure (config) {
+    Plugin.prototype.configure.call(this, config)
```

Comment:
Calling it on super instead would be cleaner and more flexible if the inheritance chain changes.